client/jetpack-cloud/sections/agency-dashboard/sites-overview/types.ts

```diff
@@ -68,6 +76,13 @@ export interface SiteNode {
 export interface StatsNode {
 	type: AllowedTypes;
 	data: SiteStats;
+	status: AllowedStatusTypes | string;
```

Comment:
I think we can keep this as `status: AllowedStatusTypes`

Reply:
Is there a reason that we wouldn't have to define the type as string?

If I remove the ` | string` then I have to add `active` as an `AllowedStatusTypes` (line 14).

Reply:
There is no use in having `AllowedStatusTypes` if we keep it as `AllowedStatusTypes | string` because there won't be proper type checking. It is similar to having only `string`.